### DISCUSSION TITLE
[BUG] Pyarmor fails to install in alpine

### DISCUSSION THREAD

**Discussion by PriOliveira** [dashingsoft/pyarmor#1161] [Q&A] [answered]
When using `python:3.11-alpine` as a base docker image, pyarmor cannot be installed successfully:

```
 > RUN pip install pyarmor==8.1.5:
#7 1.982 Collecting pyarmor==8.1.5
#7 2.138   Downloading pyarmor-8.1.5-py2.py3-none-any.whl (2.3 MB)
#7 2.476      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.3/2.3 MB 6.9 MB/s eta 0:00:00
#7 2.529 ERROR: Could not find a version that satisfies the requirement pyarmor.cli.core~=2.1.5 (from pyarmor) (from versions: none)
#7 2.529 ERROR: No matching distribution found for pyarmor.cli.core~=2.1.5

```

**Comment by jondy** [comment, marked as answer]
alpine is still not supported

**Comment by Kathir2023** [comment]
@jondy  any sample to test python flask with container.

We have plan to create docker image with pyarmor and deploy to kubernetes cluster. guide me how and where i can find some documentation. we bought group license to test docker feature

**Reply by jondy** [reply]
Sorry there is still not flask example. Just take the obfuscated script as normal python scripts with one extension module `pyarmor_runtime` to use it

**Reply by Kathir2023** [reply]
@jondy  thanks let me test, if you have any samples with pyarmor to container to my reference

**Reply by Kathir2023** [reply]
@jondy looking for your direction to solve issue

**Comment by jondy** [comment]
The latest Pyarmor 8.2.7 should support Alpine linux

**Reply by Kathir2023** [reply]
yes, facing issue with create docker image with group license for build image
